common/cli/src/bin.ts

```diff
@@ -0,0 +1,13 @@
+import { Command } from "commander";
+import { createProject } from "./createProject";
+
+const program = new Command();
+
+program
+  .command("create <projectName>")
+  .description("create project")
```

Comment:
Because of it, we don't need README because it log how to use it itself.
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/3580430/143915336-bc830adb-5ffe-4549-b3ee-04cd61eb09a9.png)